client/js/components/GameView/MoveChoiceOverlay.vue

```diff
@@ -245,6 +245,9 @@ export default {
       });
     },
     cardWidth() {
+      if (this.$vuetify.breakpoint.name === 'xs') {
+        return '100%';
+      }
```

Comment:
This feels really weird to target it in this way. If we're going to keep vuetify I would want to abstract this in to global vue helpers that work something like this:
```js
this.$isSmall = ['xs', 's'].includes(this.$vuetify.breakpoint.name);
this.$isMedium = this.$vuetify.breakpoint.name === 'md';
this.$isLarge = ['lg', 'xl'].includes(this.$vuetify.breakpoint.name);
```
or whatever the vuetify values would be so we have a single utility to update if they ever change.

Reply:
My issue with global mixins is they feel magical. It can be opaque where they come since they're not explicitly imported functions. The vuetify [breakpoint service](https://vuetifyjs.com/en/features/breakpoints/#breakpoint-service-object) actually has most of these utilities. `name` exists as a string basically so you can use `switch`statements with them, but there are also boolean values for whether the current breakpoint matches a specific size or whether it is less-than-or-equal-to it, etc.

So we could just
```suggestion
      if (this.$vuetify.breakpoint.xs) {
        return '100%';
      }
```

Reply:
I think the breakpoint service has all the utilities/shortcuts we'd want here without needing to write it as a global mixin

Reply:
That's fair, though I think some magic is ok when it eases developmental concerns. Same way that you can have something like `this.$cookie`, or `this.$logger`, etc. I think more appropriately we shouldn't need to do much at the JS level and it should be handled by the CSS side of things with some additional planning.